gettsim/jax.py

```diff
@@ -0,0 +1,277 @@
+import ast
+import functools
+import inspect
+from copy import deepcopy
+from importlib import import_module
+from typing import Dict
+
+import astor
+
+
+DEFAULT_MAPPING = {" and ": " & ", " or ": " | ", " not ": " ~"}
+
+
+def change_if_to_where_wrapper(
+    func: callable, backend: str = "np", mapping: Dict[str, str] = DEFAULT_MAPPING
```

Comment:
```suggestion
    func: callable, backend: str, mapping: Dict[str, str] = DEFAULT_MAPPING
```
I would prefer not to have a default backend.

Also, could you explain the rationale for having the DEFAULT_MAPPING as opposed to baking in the functionality? At least in the context of GETTSIM, I think it would be less confusing to just document what the function is doing rather than having a flexible interface.